schemas/sppi/v2.json

```diff
@@ -0,0 +1,175 @@
+                "maxLength": 75,
+                "examples": ["(FROM CONTAINER BERTH TO INLAND)"]
+              },
+              "service_description_4": {
+                "type": "string",
+                "description": "A separate field of the specification for the service produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 75,
+                "examples": ["CUSTOMER: BRIN YT"]
+              },
+              "service_description_5": {
+                "type": "string",
+                "description": "A separate field of the specification for the service produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 75,
+                "examples": ["PRICE PER TONNE"]
+              },
+              "unit_of_sale": {
+                "type": "string",
+                "description": "The unit of sale for the service produced by the business.",
+                "maxLength": 40,
+                "examples": ["PRICE PER MILE"]
+              },
+              "terms_of_sale_1": {
+                "type": "string",
+                "description": "A separate field of the terms of sale for the service produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 40,
+                "examples": ["CUSTOMER REF: UK CUSTOMER"]
+              },
+              "terms_of_sale_2": {
+                "type": "string",
+                "description": "A separate field of the terms of sale for the service produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 40,
+                "examples": ["PAYMENT: 25% DEPOSIT UP FRONT & FINAL"]
+              },
+              "terms_of_sale_3": {
+                "type": "string",
+                "description": "A separate field of the terms of sale for the service produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 40,
+                "examples": ["PAYMENT 28 DAYS PRIOR OR 21 DAYS IF"]
+              },
+              "terms_of_sale_4": {
+                "type": "string",
+                "description": "A separate field of the terms of sale for the service produced by the business, the contents and order of this field can change dependent on the number of characters.",
+                "maxLength": 40,
+                "examples": ["CREDIT ACCOUNT"]
+              },
+              "currency": {
+                "type": "string",
+                "description": "The currency for the service delivered.",
+                "minLength": 1,
+                "maxLength": 25,
+                "examples": ["Pound Sterling"]
```

Comment:
Spreadsheet has 'Pounds Sterling' (with an 's')
If change needed, would effect the example file as well